ui/components/component-library/avatar-token/avatar-token.js

```diff
@@ -77,23 +78,23 @@ export const AvatarToken = ({
 
 AvatarToken.propTypes = {
   /**
-   * The tokenName accepts the string to render the first letter of the AvatarToken. This will be used as the fallback display if no image url is passed to the tokenImageUrl
+   * The name accepts the string to render the first letter of the AvatarToken. This will be used as the fallback display if no image url is passed to the src
    */
-  tokenName: PropTypes.string,
+  name: PropTypes.string,
   /**
-   * The tokenImageUrl accepts the string of the image to be rendered
+   * The src accepts the string of the image to be rendered
    */
-  tokenImageUrl: PropTypes.string,
+  src: PropTypes.string,
   /**
    * The showHalo accepts a boolean prop to render the image with halo effect
    */
   showHalo: PropTypes.bool,
   /**
    * The size of the AvatarToken.
-   * Possible values could be 'SIZES.XS', 'SIZES.SM', 'SIZES.MD', 'SIZES.LG', 'SIZES.XL'
-   * Defaults to SIZES.MD
+   * Possible values could be 'AVATAR_TOKEN_SIZES.XS'(16px), 'AVATAR_TOKEN_SIZES.SM'(24px), 'AVATAR_TOKEN_SIZES.MD'(32px), 'AVATAR_TOKEN_SIZES.LG'(40px), 'AVATAR_TOKEN_SIZES.XL'(48px)
```

Comment:
Use `SIZES` here. Also I'm not sure we want to suggest it's a string? Will do a bulk PR to update that 

```suggestion
   * Possible values could be SIZES.XS(16px), SIZES.SM(24px), SIZES.MD(32px), SIZES.LG(40px), SIZES.XL(48px)
```